specification/communication/data-plane/Microsoft.CommunicationServicesSms/stable/2021-03-07/communicationservicessms.json

```
@@ -0,0 +1,199 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Communication SMS Service",
+    "description": "Azure Communication SMS Service",
+    "contact": {
+      "email": "acsdevexdisc@microsoft.com"
+    },
+    "version": "2021-03-07"
+  },
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/sms": {
+      "post": {
+        "tags": [
+          "Sms"
+        ],
+        "summary": "Sends a SMS message from a phone number that belongs to the authenticated account.",
+        "operationId": "Sms_Send",
+        "x-ms-examples": {
+          "SuccessfulSend": {
+            "$ref": "./examples/send.json"
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "SendMessageRequest",
+            "description": "Represents the body of the send message request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SendMessageRequest"
+            }
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "description": "Version of API to invoke.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "header",
+            "name": "repeatability-Request-ID",
```

Should the first letter be upper-case to match https://docs.oasis-open.org/odata/repeatable-requests/v1.0/cs01/repeatable-requests-v1.0-cs01.html#_Toc45116386 ?

```suggestion
            "name": "Repeatability-Request-ID",
```

Maybe we can ask the board about it

---

Resolved as per email communication.